include/mega/command.h

```diff
@@ -56,6 +56,11 @@ class MEGA_API Command
     // some commands can only succeed if they are in their own batch.  eg. smss, when the account is blocked pending validation
     bool batchSeparately;
 
+    // true if the command processing has been updated to use the URI v3 system, where successful state updates arrive via actionpackets.
+    bool mV3 = true;
+    bool mStringIsNotSeqtag = false;
```

Comment:
What's the purpose of this var? I think a descriptive line will help ;)
Same for `mSeqtagArray`.

Reply:
Added some comments about the purpose of these flags